coins.json

```diff
@@ -1206,6 +1206,31 @@
       "clientDocs": "https://docs.harmony.one/home/harmony-networks/harmony-network-overview/mainnet"
     }
   },
+  {
+    "id": "oasis",
+    "name": "Oasis",
+    "coinId": 474,
+    "symbol": "ROSE",
+    "decimals": 9,
+    "blockchain": "OasisNetwork",
+    "derivationPath": "m/44'/474'/0'/0'/0'",
```

Comment:
is this consistent derivation path with Lerger and Trezor?

Reply:
Yes, indeed!